core/src/main/scala/org/broadinstitute/dsde/rawls/bucketMigration/BucketMigrationService.scala

```diff
@@ -47,7 +122,7 @@ class BucketMigrationService(val dataSource: SlickDataSource, val samDAO: SamDAO
               dataAccess.multiregionalBucketMigrationQuery.getMigrationAttempts(workspace)
             }
             .map { attempts =>
-              workspace.name -> attempts.mapWithIndex(
+              workspace.toWorkspaceName.toString -> attempts.mapWithIndex(
```

Comment:
Sort of unnecessary to include the namespace here since all workspaces are in the same namespace, but I think it's clearer as the full `WorkspaceName` is globally unique while `name` isn't